iree/runtime/BUILD

```diff
@@ -4,12 +4,24 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+load("//iree:build_defs.oss.bzl", "iree_cmake_extra_content")
+
 package(
     default_visibility = ["//visibility:public"],
     features = ["layering_check"],
     licenses = ["notice"],  # Apache 2.0
 )
 
+# Generic (bare-metal) system doesn't have the file system support.
```

Comment:
We should return an unavailable/unimplemented/etc. error in API calls that use file IO rather than disable the entire `iree/runtime/` API on generic systems.

Reply:
It would need more cleanup then this. `iree/runtime`also requires `iree/hal/drivers` which depends on the task executor, so it would not work for the thread-less configuration.

Can we defer this to a further cleanup?

Reply:
Okay, but can you file an issue specifically for that, or update an existing issue to call it out?

Reply:
https://github.com/google/iree/issues/6247 filed.